__tests__/babel.test.js

```diff
@@ -0,0 +1,275 @@
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+const dirTree = require('directory-tree');
+
+const commandBuild = require('./test-utils/command-build');
+const generateFixture = require('./test-utils/generate-fixture');
+const stubNodeModule = require('./test-utils/create-stub-node-module');
+
+jest.setTimeout(15 * 1000);
+
+const tap = prom =>
+  prom.then(res => {
+    return res;
+  });
```

Comment:
I don't think I understand the purpose of this utility. How is `tap(x).then(y => {..})` different from `x.then(y => {..})`?

Reply:
Oops, my bad! This is a left over from debugging the tests. I will go ahead and remove this.